cockatrice/src/qt-json/json.cpp

```diff
@@ -269,8 +269,7 @@ QVariant Json::parseObject(const QString &json, int &index, bool &success)
         Json::nextToken(json, index);
 
         //Loop through all of the key/value pairs of the object
-        bool done = false;
-        while(!done)
+        while(true)
```

Comment:
Let's not change it. We can tell codacy to ignore the thing. This is code we vendored in - I can actually set up an ignore on the whole directory